cos_alerter/alerter.py

```diff
@@ -0,0 +1,53 @@
+import fcntl
+import json
+import tomllib
+
+import apprise
+
+
```

Comment:
**suggestion:** I think we need a module docstring here.

Reply:
See #20